packages/swap/src/routes/__tests__/v2/swap.test.ts

```diff
@@ -702,6 +734,96 @@ describe('server', () => {
       });
     });
 
+    it(`retrieves a ccm swap in ${StateV2.Failed} status (insufficient deposit amount)`, async () => {
+      const depositChannelEvent = clone(swapEventMap['Swapping.SwapDepositAddressReady']);
+      depositChannelEvent.args.ccmParams = {
+        gasBudget: '50000000',
+        message: '0xd3adc0de',
+      };
+      depositChannelEvent.args.destinationAsset.__kind = 'Usdc';
+      depositChannelEvent.args.sourceAsset.__kind = 'Usdc';
+
+      const requestedEvent = clone(swapEventMap['Swapping.SwapRequested']);
+      requestedEvent.args = {
+        ...requestedEvent.args,
+        inputAsset: { __kind: 'Usdc' },
+        outputAsset: { __kind: 'Usdc' },
+        __kind: 'Ccm',
+        outputAddress: {
+          value: '0x2afba9278e30ccf6a6ceb3a8b6e336b70068f045c666f2e7f4f9cc5f47db8972',
+          __kind: 'Eth',
+        },
+        ccmDepositMetadata: {
+          sourceChain: {
+            __kind: 'Ethereum',
+          },
+          sourceAddress: {
+            value: '0x2afba9278e30ccf6a6ceb3a8b6e336b70068f045c666f2e7f4f9cc5f47db8972',
+            __kind: 'Eth',
+          },
+          channelMetadata: {
+            message:
+              '0x000000000000000000000000000000000000000000000000000000000000004000000000000000000000000067ff09c184d8e9e7b90c5187ed04cbfbdba741c8000000000000000000000000000000000000000000000000000000000000000c6461676f61746973686572650000000000000000000000000000000000000000',
+            gasBudget: '50000000',
+            cfParameters: '0x',
+          },
+        },
+      };
+
+      const finalizedEvent = clone(swapEventMap['EthereumIngressEgress.DepositFinalised']);
+      finalizedEvent.args.asset.__kind = 'Usdc';
+
+      await processEvents(
+        [
+          depositChannelEvent,
+          requestedEvent,
+          swapEventMap['Swapping.CcmFailed'],
+          finalizedEvent,
+          swapEventMap['Swapping.SwapRequestCompleted'],
+        ],
+        [
+          {
+            id: '0000000092-000399-02fea',
+            call: {
+              value: {
+                __kind: 'process_deposits',
```

Comment:
what exactly is this event ? 🧐

Reply:
its not an event, but a call - the `ccmFailed` handler fetches this from the indexer to get some additional information: https://github.com/chainflip-io/chainflip-sdk-monorepo/blob/7d97a8691fa14243ae1f234e05b66e4610b96897/packages/swap/src/event-handlers/ccmFailed.ts#L46-L48